build.gradle

```diff
@@ -372,8 +399,42 @@ task copyHtml(type: Copy) {
 // TODO: Temporary workaround for new exes / JRE bundling not quite being officially ready yet
 distZip.exclude("*.x*.exe")
 
+task unpackZipForNSIS(type: Copy) {
+    dependsOn distZip
+
+    def zipFile = file("${buildDir}/distributions/TerasologyLauncher.zip")
+    def tmpDir = file("${buildDir}/tmp/distNSIS")
+    from zipTree(zipFile)
+    into tmpDir
+}
+
+task renderNSISScript(type: Copy) {
```

Comment:
> So I used "render" which is commonly used with templates, but I can change it if you want.

Thanks for the explanation. I believe your words on the template thing here 😉 Leave it as `render`